app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -144,8 +144,13 @@ fun PledgedProjectsOverviewScreen(
                             showBadge = it.showBadge,
                             onActionButtonClicked = { },
                             onSecondaryActionButtonClicked = {
-                                confirmedAddress = it.shippingAddress
-                                openConfirmAddressAlertDialog.value = true
+                                when (it.viewType) {
+                                    PPOCardViewType.CONFIRM_ADDRESS -> {
```

Comment:
Amending this to only call this particular action when the card type is confirm address.